content/docs/introduction/faq.md

```diff
@@ -30,6 +30,22 @@ The Alertmanager cannot currently be made highly available, but this is a goal.
 
 Most Prometheus components are written in Go. Some are also written in Java and Ruby.
 
+### Why do you pull rather than push?
+
+Pulling over HTTP offers a number of advantages:
+
+- You can run your monitoring on your laptop when developing changes.
+- You can more easily tell if a target is down.
+- You can manually go to a target and inspect its health with a web browser.
+
+Overall we believe that push is slightly better than pull, but it should not be
```

Comment:
"pushing is slightly better than pulling" would be more grammatically correct if we embed it into the sentence like this